src/pspm_sf_dcm.m

```diff
@@ -18,19 +18,23 @@
 %       .yhat:  fitted time series
 %      .model:  information about the DCM inversion
 % ● Arguments
-%         scr:  skin conductance epoch (maximum size depends on computing
-%               power, a sensible size is 60 s at 10 Hz)
-%          sr:  sampling rate in Hz
-%     options:  options structure
-%  .threshold:  threshold for SN detection (default 0.1 mcS)
-%      .theta:  a (1 x 5) vector of theta values for f_SF
-%               (default: read from pspm_sf_theta)
-%      .fresp:  frequency of responses to model (default 0.5 Hz)
-%    .dispwin:  display progress window (default 1)
-%   .dispsmallwin:
-%               display intermediate windows (default 0);
-%    .missing:  index of missing values to ignore
-%   .missingthresh:
+%   ┌──────model
+%   │ ▶︎ Mandatory
+%   ├───────.scr:  skin conductance epoch (maximum size depends on computing
+%   │              power, a sensible size is 60 s at 10 Hz)
+%   ├────────.sr:  sampling rate in Hz
+%   │ ▶︎ Optional
+%   └.missing_data: the datafile of missing epochs
```

Comment:
this should be changed - it is an index?

Reply:
This is my fault. This should be the index of missing values / epochs. I have now corrected this line and checked other relevant comments.

Reply:
The line does not seem to be updated?

Reply:
It is a little strange, my side shows it is updated, please let me check what happened...
<img width="681" alt="image" src="https://github.com/bachlab/PsPM/assets/3895146/a80fba28-760f-4638-89b9-7d2e553c5f1a">